tools/experiments/experiments.js

```diff
@@ -366,10 +397,10 @@ function showConfirmation_(message, callback) {
 }
 
 /**
- * Loads the AMP_CONFIG objects from whatever the v0.js is that the
- * user has (depends on whether they opted into canary or RC), so that
- * experiment state can reflect the default activated experiments.
- * @return {*} TODO(#23582): Specify return type
+ * Loads the AMP_CONFIG objects from whatever the v0.js is that the user has
+ * (depends on whether they opted into beta/experimental), so that experiment
```

Comment:
For completeness, include RTV in the comment.

> (depends on whether they opted into beta, experimental, or a specific RTV)

Reply:
Done